lib/adapters/role_adapter.spec.ts

```diff
@@ -30,10 +30,10 @@ describe("maintainerAdapter", () => {
   it("returns left path for invalid date", () => {
     // @ts-ignore
     role.createdOn = "Invalid date"
-    expect(E.isLeft(roleAdapter(role))).toBe(true)
+    expect(E.isLeft(roleAdapter(role!))).toBe(true)
```

Comment:
again, prefer type casting or nullish coalescing over null assertion

Reply:
i assume that's because `role.createdOn` is a string and not a `Date`. since this is only a test file, i guess it doesn't matter much